packages/twenty-server/src/main.ts

```diff
@@ -68,7 +86,17 @@ const bootstrap = async () => {
     app.use(session(getSessionStorageOptions(environmentService)));
   }
 
-  await app.listen(process.env.PORT ?? 3000);
+  await app.listen(serverUrl.port, serverUrl.hostname);
+
+  const url = new URL(await app.getUrl());
+
+  // prevent ipv6 issue for redirectUri builder
+  url.hostname = url.hostname === '[::1]' ? 'localhost' : url.hostname;
+
+  ServerUrl.set(url.toString());
```

Comment:
Make API_URL mandatory (maybe with default in environment.variable or in .env.example)